.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org 
```

Comment:
This is necessary to prevent npm installs with private repos from overwriting package.lock